quickwit-cli/src/index.rs

```diff
@@ -779,6 +779,12 @@ pub async fn create_index_cli(args: CreateIndexArgs) -> anyhow::Result<()> {
     let metastore = metastore_uri_resolver
         .resolve(&quickwit_config.metastore_uri())
         .await?;
+
+    // Check if storage uri is valid
+    metastore_uri_resolver
+        .resolve(&quickwit_config.default_index_root_uri())
```

Comment:
An index URI is not necessarily `{default_index_root_uri}/{index_id}` when overridden in the index config. We need to check that too.

Reply:
Good point, I added a check for it